source/proof-of-concept-guide/poc-vulnerability-detection.rst

```diff
@@ -51,12 +51,65 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
 
       $ sudo systemctl restart wazuh-manager
 
-Visualize the alerts
---------------------
+Visualizing the alerts
```

Comment:
What is shown in the images below are indexed vulnerabilities among other sections. Vulnerability alerts heading is related to the "events" section. Couldn't that confuse a little bit the readers?

Reply:
Done!